verticapy/jupyter/extensions/chart_magic.py

```diff
@@ -192,30 +192,427 @@ def chart_magic(
+    * Drawing graphics
+    * Exporting to HTML
+    * Using variables
+    * Using SQL files
+
+    Setting up the environment
+    ==========================
+
+    If you don't already have one, create a new connection:
+
+    .. code-block:: python
+
+        import verticapy as vp
+
+        # Save a new connection
+        vp.new_connection({"host": "10.211.55.14",
+                           "port": "5433",
+                           "database": "testdb",
+                           "password": "XxX",
+                           "user": "dbadmin"},
+                           name = "VerticaDSN")
+
+    Otherwise, to use an existing connection:
+
+    .. code-block:: python
+
+        vp.connect("VerticaDSN")
+
+    Load the chart extension:
+
+    .. ipython:: python
+        :suppress:
+
+        import verticapy as vp
+
+    .. ipython:: python
+        :suppress:
+
+        %load_ext verticapy.hchart
+
+    Run the following to load some sample datasets. Once loaded, these datasets are
+    stored in the 'public' schema. You can change the target schema with the
+    'schema' parameter:
+
+    .. ipython:: python
+
+        from verticapy.datasets import load_titanic, load_amazon, load_iris
+
+        titanic = load_titanic()
+        amazon = load_amazon()
+        iris = load_iris()
+
+    Use the set_option function to choose your desired plotting library:
+
+    .. ipython:: python
+
+        vp.set_option("plotting_lib","plotly")
+
+    Drawing graphics
+    ================
+
+    The following examples draw various responsive charts from SQL queries:
+
+    .. code-block:: python
+
+        %chart -k pie -c "SELECT pclass, AVG(age) AS av_avg FROM titanic GROUP BY 1;"
+
+    .. ipython:: python
+        :suppress:
+
+        %%chart -k pie
+        SELECT pclass, AVG(age) AS av_avg FROM titanic GROUP BY 1;
+
+    .. ipython:: python
+        :suppress:
+
+        pie_chart = _
+        pie_chart.write_html("figures/jupyter_extensions_chart_magic_chart_magic.html")
+
+    .. raw:: html
+        :file: SPHINX_DIRECTORY/figures/jupyter_extensions_chart_magic_chart_magic.html
+
+    .. code-block:: python
+
+        %%chart -k line
+        SELECT
+            date,
+            AVG(number) AS number
+        FROM amazon
+        GROUP BY 1;
+
+    .. ipython:: python
+        :suppress:
+
+        %%chart -k line
+        SELECT
+            date,
+            AVG(number) AS number
+        FROM amazon
+        GROUP BY 1;
+
+    .. ipython:: python
+        :suppress:
+
+        line_chart = _
+        line_chart.write_html("figures/jupyter_extensions_chart_magic_chart_magic_2.html")
+
+    .. raw:: html
+        :file: SPHINX_DIRECTORY/figures/jupyter_extensions_chart_magic_chart_magic_2.html
+
+    .. code-block:: python
+
+        %%chart -k pearson
+        SELECT
+            *
+        FROM titanic;
+
+    .. ipython:: python
+        :suppress:
+
+        %%chart -k pearson
+        SELECT * FROM titanic;
+
```

Comment:
Heatmap
^^^^^^^

Reply:
@tconsigny heat map section missing

Reply:
This example is already sectioned as Correlation Matrix. Should it be Heatmap instead?